scripts/test_parse_directory.py

```diff
@@ -39,7 +39,9 @@
 argparser.add_argument(
     "--skip-actions", action="store_true", help="Suppress code emission for rule actions",
 )
-argparser.add_argument("-t", "--tree", action="count", help="Compare parse tree to official AST")
+argparser.add_argument(
+    "-t", "--tree", action="count", help="Compare parse tree to official AST", default=0
```

Comment:
Curious -- isn't the default 0 for `action="count"`? It always worked for me.

Reply:
Nope! The default here is `None`, according to the docs.